lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module

```diff
@@ -277,9 +277,29 @@ function dosomething_campaign_run_form_campaign_run_node_form_alter(&$form, &$fo
   // Add title help text.
   $form['title']['#description'] = t('The title is not user facing, it just helps when searching for content. Suggestion: Use campaign name and year campaign ended (i.e. Comeback Clothes 2014).');
 
+  // Alter the Run Date text
+  $form['#after_build'][] = 'dosomething_campaign_run_after_build';
+
   drupal_add_css(drupal_get_path('module', 'dosomething_campaign_run') . '/campaign_run_node.css');
 }
 
+/**
+ * Customize the Run Date form
+ */
+function dosomething_campaign_run_after_build($form, &$form_state) {
+  if (!isset($form['nid']['#value'])) {
+    $form['field_run_date'][LANGUAGE_NONE][0]['show_todate']['#title'] = t('Select End Date');
+  }
+
+  // Make sure to account for all the language options as well
+  $prefixes = dosomething_global_get_language_prefix_list();
+  foreach ($prefixes as $lang) {
+    $form['field_run_date'][$lang][0]['show_todate']['#title'] = t('Select End Date');
```

Comment:
I think alsoooo you want to check if the translation exsits, check the [dblog](http://dev.dosomething.org:8888/admin/reports/dblog) for what I bet is some lovely notices!